make-cider.nix

```diff
@@ -0,0 +1,26 @@
+{
+  nixpkgs,
+  darwin,
```

Comment:
@gtrunsec passing darwin like this in the face of the `bee` module seems a bit bespoke and fix the interface at the wrong layer.

Can't we complement the bee module to add a darwin option, instead?

Reply:
Yes, we can. Why not. I just did a minimal(ditry) patch there that doesn't need to change bee.